src/features/search/components/SearchListHeader/SearchListHeader.native.test.tsx

```diff
@@ -348,7 +352,10 @@ describe('<SearchListHeader />', () => {
       expect(screen.queryByText('2 résultats')).toBeNull()
     })
 
-    it('should not render venue items when there are not venues', () => {
+    it('should not render venue items when we search from venues', () => {
```

Comment:
Je préfère `should not render venue items when location type is VENUE` mais à toi de voir je valide quand même